service/src/main/java/tds/session/configuration/SessionServiceProperties.java

```diff
@@ -0,0 +1,34 @@
+package tds.session.configuration;
+
+import com.google.common.base.Preconditions;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties("session-service")
+public class SessionServiceProperties {
+    private String examUrl = "";
+
+    /**
+     * @return url for the exam microservice
+     */
+    public String getExamUrl() {
+        return examUrl;
+    }
+
+    public void setExamUrl(String examUrl) {
+        this.examUrl = removeTrailingSlash(Preconditions.checkNotNull(examUrl, "examUrl cannot be null"));
+    }
+
+    /**
+     * If there is a trailing slash at the end of a supplied URL, remove it.
+     *
+     * @param url The URL being trimmed
+     * @return The url without a trailing slash
+     */
+    private String removeTrailingSlash(String url) {
```

Comment:
Good idea - I've added a `UrlUtils` to `tds.common.web.utils`, which might not be the best spot for it.  I put it in a `web.utils` package because URLs tend to be web-specific.  I could easily see this class landing in a `tds.common.utils` package.  I will update this comment w/a link to the PR for *TDS_Common* when I have one put up.